Esp32 firmware build error on Debian and Raspberry

### Expected behavior
build without errors

### Actual behavior

When building the esp32 firmware, the following error occurs during make:

```bash
/home/pi/esp32/nodemcu-firmware-esp32/components/modules/node.c: In function 'node_info':
/home/pi/esp32/nodemcu-firmware-esp32/components/modules/node.c:475:45: error: expected expression before ',' token
  475 |       add_int_field(L,    BUILDINFO_LFS_SIZE,   "lfs_size");
...
```

### Test code

make

### NodeMCU startup banner
```
NodeMCU ESP32 build 2024-12-19 08:01 powered by Lua 5.3.5 [5.3-int32-singlefp] on IDF v5.3.2
cannot open init.lua: No such file or directory
```

### Hardware
Esp32 devkit

--- a/components/modules/CMakeLists.txt
+++ b/components/modules/CMakeLists.txt
@@ -254,7 +254,7 @@ execute_process(
 COMMAND grep "^lfs,.*" ${PROJECT_DIR}/components/platform/partitions.csv
 COMMAND cut -d, -f5
 COMMAND tr -d " "
-COMMAND awk "{print strtonum( $1 )}"
+COMMAND xargs printf "%d"
 OUTPUT_STRIP_TRAILING_WHITESPACE
 OUTPUT_VARIABLE buildinfo_lfs_size
 )

